tasks/tests/unit/test_process_flakes.py

```diff
@@ -51,10 +51,51 @@ def add_test_instance(self, c, outcome=TestInstance.Outcome.PASS.value):
             outcome=outcome,
             test=self.test_map[self.test_count],
         )
-        print(self.test_map[self.test_count].id)
         ti.save()
+
+        rollup, _ = DailyTestRollup.objects.get_or_create(
+            repoid=self.repo.repoid,
+            date=dt.date.today(),
+            test=self.test_map[self.test_count],
+            branch=c.branch,
+            defaults={
+                "pass_count": 0,
+                "skip_count": 0,
+                "fail_count": 0,
+                "flaky_fail_count": 0,
+                "avg_duration_seconds": 0.0,
+                "last_duration_seconds": 0.0,
+                "latest_run": dt.datetime.now(),
+                "commits_where_fail": [],
+            },
+        )
+
+        match outcome:
+            case TestInstance.Outcome.PASS.value:
+                rollup.pass_count += 1
+            case TestInstance.Outcome.SKIP.value:
+                rollup.skip_count += 1
+            case _:
```

Comment:
is this identical to "default" in other languages?

Reply:
yep